augly/image/functional.py

```diff
@@ -1058,6 +1078,7 @@ def overlay_emoji(
     image = imutils.validate_and_load_image(image)
 
     func_kwargs = imutils.get_func_kwargs(metadata, locals())
+    src_mode = image.mode
```

Comment:
Same issue here as below! `src_mode` isn't getting passed into `ret_and_save_image()` because it isn't called in this augmentation. That's a mistake, so please change line 1102 below to `return imutils.ret_and_save_image(aug_image, output_path, src_mode)`

Reply:
wait, @zpapakipos -- i don't think it's needed to have these extra calls to `ret_and_save_image`. this function, `overlay_emoji` calls within it `overlay_image` to do the actual overlaying of the emoji onto the image. thus, as long as `overlay_image` handles the modes correctly, *we do not need to add these extra function calls*. 

tl;dr you can remove `src_mode = image.mode` and the ret and save call from this function as they are already handled by `overlay_image`. -- let me know if you see what i mean @zpapakipos 

sorry for the extra work! if you've done this anywhere else, please undo those changes too (ill try to comment where i find them)